packages/core/src/components/button/button-mixin.scss

```
@@ -94,14 +94,12 @@
   flex-wrap: nowrap;
-
-  overflow: hidden;
-
   align-items: center;
   justify-content: center;
 
   width: 100%;
   height: 100%;
+  overflow: var(--ix-button-inner-overflow, hidden);
 
   padding: var(--ix-button-padding, 0 0.5rem);
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)